addon/mixins/in-viewport.js

```diff
@@ -145,8 +145,8 @@ export default Mixin.create({
 
       const enterCallback = () => {
         const isTearingDown = this.isDestroyed || this.isDestroying;
-        const viewportEntered = element.getAttribute('data-in-viewport-entered');
-        if (!isTearingDown && viewportSpy || viewportEntered) {
+        const viewportEntered = JSON.parse(element.getAttribute('data-in-viewport-entered'));
```

Comment:
Thank you!!!  possible we can just do `(viewportEntered === "true")`?

Reply:
Yeah that might be a good idea to be a little more explicit...

Reply:
👍